components/src/web-components/display/mutation-comparison-component.stories.ts

```diff
@@ -127,3 +129,19 @@ export const Default = {
         },
     },
 };
+
+export const VennDiagram: StoryObj<MutationComparisonProps> = {
+    ...Default,
+    play: async ({ canvasElement }) => {
+        const canvas = await withinShadowRoot(canvasElement, 'gs-mutation-comparison-component');
+
+        await waitFor(() => expect(canvas.getByLabelText('Venn', { selector: 'input' })).toBeInTheDocument());
+
+        canvas.getByLabelText('Venn', { selector: 'input' }).click();
+        await waitFor(() =>
+            expect(
+                canvas.getByText('You have no elements selected. Click in the venn diagram to select.'),
+            ).toBeInTheDocument(),
```

Comment:
```suggestion
            ).toBeVisible(),
```

If I remove the `click` above, the test is still green. With `toBeVisible` it's read without the click. (The Venn tab content is actually in the document, but not visible.)

Reply:
It seems that `userEvent` does not the same as `fireEvent`. I now use `fireEvent`.